packages/inventory/src/components/InventoryList/index.tsx

```diff
@@ -53,7 +53,7 @@ const InventoryList = (props: InventoryListProps) => {
   useEffect(() => {
     // api call to get inventory by id
     const serve = new OpenSRPService(
-      `${GET_INVENTORY_BY_SERVICE_POINT}${servicePointId}`,
+      `${GET_INVENTORY_BY_SERVICE_POINT}${servicePointId}?returnProduct=true`,
```

Comment:
list has an argument that takes search params as an object, we should use that.

Reply:
@p-netm 
Followed this, 
https://github.com/OpenSRP/opensrp-server-web/issues/711#issuecomment-775880968

Reply:
yeah that's ok, am asking about the code implementation itself, the opensrpService list method has a parameter that allows you to pass an object representing the urlSearchParams as an object instead of creating the url yourself

Reply:
oh, got your point.